operating_systems.go

```diff
@@ -6,6 +6,25 @@ package user_agent
 
 import "strings"
 
+// Represents full information on the operating system extracted from the user agent.
+type OSInfo struct {
+	// Full name of the operating system. This is identical to the output of ua.OS()
+	FullName string
+
+	// Name of the operating system. This is sometimes a shorter version of the
+	// operating system name, e.g. "Mac OS X" instead of "Intel Mac OS X"
+	Name string
+
+	// Operating system version, e.g. 7 for Windows 7 or 10.8 for Max OS X Mountain Lion
+	Version string
+
+	// Same as ua.Localization()
+	Localization string
+
+	// Same as ua.Platform()
+	Platform string
```

Comment:
I'm not entirely sure why we'd want either `Localization` and `Platform` here. They already have their own getters which are plain and convenient. What do you think ?